sys/net/routing/rpl/rpl_dodag.c

```diff
@@ -207,13 +207,9 @@ rpl_parent_t *rpl_find_parent(ipv6_addr_t *address)
 
 void rpl_delete_parent(rpl_parent_t *parent)
 {
-    rpl_dodag_t *my_dodag = rpl_get_my_dodag();
-
-    if ((my_dodag != NULL) && rpl_equal_id(&my_dodag->my_preferred_parent->addr,
```

Comment:
@cgundogan I assume that `rpl_equal_id(...)` is obsolete/wrong in the RPL implementation since we now provide multiple DODAGs?